src/site/stages/build/process-cms-exports/transformers/paragraph-wysiwyg.js

```diff
@@ -0,0 +1,20 @@
+const { getDrupalValue, removeLinebreaks } = require('./helpers');
+
+function wysiwygTransform(entity) {
+  const { fieldWysiwyg } = entity;
+
+  const transformed = {
+    entity: {
+      entityType: 'paragraph',
+      entityBundle: 'wysiwyg',
+
+      fieldWysiwyg: {
+        processed: removeLinebreaks(getDrupalValue(fieldWysiwyg)),
```

Comment:
For what it's worth, we'll also need to translate Unicode code points into Unicode characters (e.g. `\u2014` -> `—`).